xds/src/main/java/io/grpc/xds/EnvoyProtoData.java

```diff
@@ -195,6 +196,14 @@ public String toString() {
     private final int loadBalancingWeight;
     private final boolean isHealthy;
 
+    @VisibleForTesting
+    LbEndpoint(String address, int port, int loadBalancingWeight, boolean isHealthy) {
+      this(
+          new EquivalentAddressGroup(
+              ImmutableList.<SocketAddress>of(new InetSocketAddress(address, port))),
+          loadBalancingWeight, isHealthy);
+    }
+
```

Comment:
Not required for this PR, we can remove the other constructor.

Reply:
Sure. I remember the other constructor was used for compatibility purposes in [`LocalityStoreTest`](https://github.com/grpc/grpc-java/blob/eb21c646b239fdf00e25abf5eaa3b7b523f5cf05/xds/src/test/java/io/grpc/xds/LocalityStoreTest.java#L207-L214). We need to migrate that test.